cluster/config-defaults.yaml

```diff
@@ -386,4 +386,4 @@ spotio_ocean_controller_memory: "512Mi"
 kubernetes_event_logger_enabled: "true"
 
 # enable/disable stackset validation (preserveUnknownFields)
-stackset_validation_enabled: "false"
+stackset_validation_enabled: "true"
```

Comment:
Note, this is disabled in cluster-registry for all clusters, but enabled and tested in e2e.